### PR TITLE
Footer section

### PR DESCRIPTION
**Pull request by MostafaBalata**
screen shot 2017-08-16 at 2 37 10 pm

### CODE REVIEW

**Comment by krzkaczor** [comment]
Can u ask Ola for proper images?

**Comment by MostafaBalata** [comment]
screen shot 2017-08-17 at 10 44 52 am

**Comment by krzkaczor** [comment]
@MostafaBalata one more thing. Each one of these links should point to the article on given site. I think they are somewhere on `neufund.org` .

**Comment by MostafaBalata** [comment]
screen shot 2017-08-18 at 2 39 50 pm

**Comment by MostafaBalata** [comment]
screen shot 2017-08-18 at 2 40 37 pm

File: page/partials/footer.ejs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="footer-container">
+    <div id="footer-hex" class="hexagon-wrapper">
+        <div class="hexagon hexagon-white-green"></div>
+    </div>
+    
+    <div class="section footer">
+        <ul>
+            <li><img src="../assets/techcrunchlogo2013.png" /></li>
```
Comment (krzkaczor) [inline page/partials/footer.ejs]: all these images should be links to appropriate articles

File: page/sass/common.scss [hunk 2]
```diff
@@ -74,4 +74,47 @@ body{
 
 .person-block{
     cursor: pointer;
+}
+
+.hexagon-blue {
```
Comment (krzkaczor) [inline page/sass/common.scss]: use nested styles for everything hexaghons related!

File: page/partials/footer.ejs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            <li><a target="_blank" href="https://techcrunch.com/2017/01/17/neufund/" ><img src="../assets/techcrunchlogo2013.png" /></a></li>
+            <li><a target="_blank" href="http://www.geektime.com/2017/01/17/berlin-cryptocurrency-hybrid-vc-neufund-raises-e2m-to-upend-startup-investing-as-we-know-it/" ><img src="../assets/gt-conv-logo.png" /></a></li>       
+            <li><a target="_blank" href="https://cointelegraph.com/news/icos-are-the-future-of-funding-says-co-founder-of-platform-that-just-raised-2-mln" ><img src="../assets/cointelegraph-1.png" /></a></li>        
+            <li><a target="_blank" href="http://www.the-blockchain.com/2017/01/21/berlins-neufund-blockchain-fundraising-platform-raises-2-million/" ><img src="../assets/logo-300x77.png" /></a></li>        
+            <li><a target="_blank" href="http://www.pymnts.com/blockchain/2017/blockchain-tracker-bridging-gaps/" ><img src="../assets/pymnts-1.png" /></a></li>
+        </ul>
+    </div>
+    <div class="footer-bottom">
+        <span class="hander">@Fifth Force</span>
+        <span class="email">neufund.org</span>
```
Comment (krzkaczor) [inline page/partials/footer.ejs]: this should be link (override styles)

File: page/partials/footer.ejs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <span class="email">icbm@neufund.org</span>
```
Comment (krzkaczor) [inline page/partials/footer.ejs]: this should me a mailto: link as well

**Pull request merged**